tensorflow_addons/optimizers/tests/lookahead_test.py

```diff
@@ -19,6 +19,7 @@
 import tensorflow as tf
 
 from tensorflow_addons.optimizers import Lookahead
+from tensorflow.python.keras.mixed_precision.experimental import policy
```

Comment:
One of the check is failing cause our policy doesn't let us to import this namespace but `policy_scope` is not exposed and in many parts there are `thread_local`  todos like in https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/mixed_precision/experimental/policy.py#L484

Reply:
See also https://github.com/tensorflow/tensorflow/pull/40475

Reply:
Seeing as we have no public API to make this thread local, I'm in favor of including this test case and adding the file to the list of allowed private usage:
https://github.com/tensorflow/addons/blob/master/tools/testing/source_code_test.py#L86

This has been confirmed by the user to solve the problem, and the private usage is not called in the optimizer source code, only the test case. Seems counterproductive to not include the test. If https://github.com/tensorflow/tensorflow/pull/40475 or some variant merges we can remove it. @gabrieldemarmiesse thoughts?